<launch>
    <node pkg="depthimage_to_laserscan" type="depthimage_to_laserscan" name="depthimage2laserscan" >
        <param name="scan_height" value="3"/>
        <param name="output_frame_id" value="camera_link"/>
        <remap from="image" to="/camera/depth_registered/image_raw" />
    </node>
</launch>